### PR TITLE
icons-into-json-file

### PR DESCRIPTION
**Pull request by cathysiller**
JIRA: care-athon 2018

### LGTM's
- [x] Dev LGTM
- [x] Zoom LGTM

### CODE REVIEW

**Comment by HelixUI** [comment]
Deploy preview for *helix-ui* processing.

Built with commit e0ee20aeb4edde66b7ca139244c73a490faf4e9b

https://app.netlify.com/sites/helix-ui/deploys/5be4c7ae05c41756f4be5e02

File: bin/buildIcons.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const globby = require('globby');
+const path = require('path');
+const iconFolder = path.resolve(__dirname, '../src/helix-ui/icons/')
+
+const Icons = {};
+globby.sync('*.svg', { cwd: iconFolder }).forEach(file => {
+    let filename = path.join(iconFolder, file)
```
Comment (CITguy) [inline bin/buildIcons.js]: * rename variable to `absFilePath`
* missing semicolon

File: bin/buildIcons.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const globby = require('globby');
+const path = require('path');
+const iconFolder = path.resolve(__dirname, '../src/helix-ui/icons/')
```
Comment (CITguy) [inline bin/buildIcons.js]: missing semicolon

File: bin/buildIcons.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+const Icons = {};
+globby.sync('*.svg', { cwd: iconFolder }).forEach(file => {
+    let filename = path.join(iconFolder, file)
+    let data = fs.readFileSync(filename, { encoding: 'utf8' });
+    Icons[file] = data;
```
Comment (CITguy) [inline bin/buildIcons.js]: `file` should be the basename of the svg file. i.e. `account.svg` should set `Icons['account']`, etc.

**Pull request merged**